lib/src/cupertino/cupertino_controls.dart

```diff
@@ -762,6 +767,9 @@ class _CupertinoControlsState extends State<CupertinoControls>
     final skip =
         (_latestValue.position + const Duration(seconds: 15)).inMilliseconds;
     controller.seekTo(Duration(milliseconds: math.min(skip, end)));
+    Future.delayed(const Duration(milliseconds: 1000), () {
```

Comment:
Same here. Add comments above this line explaining why there needs to be a one second delay before setting the playback speed.

That way, any future dev beyond yourself will be better able to understand why its being done this way.

Reply:
Added comments.